### ISSUE TITLE
torch.lgamma is undocumented

### ISSUE THREAD

**Issue by vishwakftw** [pytorch/pytorch#27783]
## 📚 Documentation

While working on fixing #25347, I found out that `torch.lgamma` appears to be undocumented.

```
>>> import torch
>>> help(torch.lgamma)
Help on built-in function lgamma:

lgamma(...)
```

cc @ezyang @gchanan @zou3519

**Comment by nightfox0909** [comment]
Love to help, but I'm new. Could you help me with a pointer ?

**Comment by vishwakftw** [comment]
@nightfox0909 sorry, this is being fixed in #27812
